src/app/shell/personal-cabinet/provider/create-achievement/create-achievement.component.html

```diff
@@ -126,7 +129,7 @@ <h3 class="wrapper-title">{{ 'FORMS.HEADERS.ADD_ACHIEVEMENT' | translate }}</h3>
           <mat-datepicker-toggle matSuffix [for]="picker"></mat-datepicker-toggle>
           <mat-datepicker #picker></mat-datepicker>
         </mat-form-field>
-        <app-validation-hint [validationFormControl]="AchievementFormGroup.get('achievementDate')"> </app-validation-hint>
+        <app-validation-hint [validationFormControl]="AchievementFormGroup.get('achievementDate')" minMaxDate="true"> </app-validation-hint>
```

Comment:
Here, you pass minMaxDate as a "true" string, not as a boolean variable. In this case, use the template expression
`[minMaxDate]="true"`